.github/pull_request_template.md

```diff
@@ -86,7 +86,19 @@ Note that GitHub prefixes anchor names in markdown with "user-content-".
 
 #### Should this PR be included in the next patch release?
 
+`Yes` should be selected for bug fixes, documentation updates, and other small changes. `No` should be selected for new features and larger changes or if there is no recent minor/patch release. If you're unsure about the release classification of this PR, leave this unchecked to let the maintainers decide.
+
+<details>
+<summary>What is a minor/patch release?</summary>
+
+- Minor release: a release that increments the second part of the version number (e.g., 1.2.0 -> 1.3.0)
```

Comment:
Can we explain what usually goes into a minor release? E.g. bug fixes, doc updates, *and* new features?